File: backend/B2BApplication/DTO/DashboardDto.cs
namespace B2BApplication.DTO
{
    public class DashboardDto
    {
        public int nbCommandeClient { get; set; }
        public int nbCommandeSociete { get; set; }
        public int nbRetourClient { get; set; }
        public int nbRetourSociete { get; set; }
        public int nbPanier { get; set; }

        public int nbWishlist { get; set; }
        public int nbCompte { get; set; }
    }
}

File: frontend/src/app/models/dashboardDto.ts
export class DashboardDto{
    nbCommandeClient:number
    nbCommandeSociete:number
    nbRetourClient:number
    nbRetourSociete:number
    nbPanier:number
    nbWishlist:number
    nbCompte:number
    constructor( nbCommandeClient:number,nbCommandeSociete:number,nbRetourClient:number
        ,nbRetourSociete:number,nbPanier:number,nbWishlist:number,nbCompte:number){
this.nbCommandeClient=nbCommandeClient
this.nbCommandeSociete=nbCommandeSociete
this.nbRetourClient=nbRetourClient
this.nbRetourSociete=nbRetourSociete
this.nbPanier=nbPanier
this.nbWishlist=nbWishlist
this.nbCompte=nbCompte
    }
}
